src/main/java/uk/gov/companieshouse/service/dissolution/DissolutionPatcher.java

```diff
@@ -65,6 +66,14 @@ public DissolutionPatchResponse addDirectorApproval(String companyNumber, String
         return this.responseMapper.mapToDissolutionPatchResponse(dissolution);
     }
 
+    public DissolutionPatchResponse updateSignatory(String companyNumber, DissolutionPatchDirectorRequest body, String officerId) throws DissolutionNotFoundException, DirectorNotFoundException {
+        final Dissolution dissolution = this.repository.findByCompanyNumber(companyNumber).orElseThrow(DissolutionNotFoundException::new);
+
+        this.updateSignatory(body, dissolution, officerId);
+
+        return this.responseMapper.mapToDissolutionPatchResponse(dissolution);
```

Comment:
Hmmmm, I wonder if we should be reusing the same model or creating a new DissolutionDirectorPatchResponse? Yes they currently contain the same values, but should we model them separately so that we can modify them in isolation for one another? Consider using inheritance if so for the response models.